Dockerfile

```diff
@@ -57,6 +58,6 @@ COPY scm-source.json /scm-source.json
 WORKDIR ${HOME}
 USER ${USER}
 
-EXPOSE 2181 2888 3888 8181 8778
+EXPOSE 2181 2888 3888 8181 8778 8779
```

Comment:
Can you please add port mapping here: https://github.com/zalando-nakadi/exhibitor-appliance/blob/master/exhibitor-appliance.yaml#L51